spring-cloud-sleuth-zipkin-stream/src/main/java/org/springframework/cloud/sleuth/zipkin/stream/ZipkinMessageListener.java

```diff
@@ -196,6 +191,13 @@ public Cloud cloud() {
 			return new CloudFactory().getCloud();
 		}
 
+		// TODO: what config shoud we use? zipkin uses zipkin.collector.sample-rate
+		@Bean
+		TraceIdSampler traceIdSampler() {
```

Comment:
@joshlong @dsyer so we need a config key for the collector's sample rate (ex. 0.7 means keep 70%). What should that be? Once we've decided, I can finish this up